docs/csharp/tutorials/console-teleprompter.md

```diff
@@ -78,12 +78,8 @@ namespace TeleprompterConsole
 ```
 
 ## Reading and Echoing the File
-The first feature to add is to read a text file, and display all that text
-to the console. First, let’s add a text file. Copy the 
-[sampleQuotes.txt](https://raw.githubusercontent.com/dotnet/docs/master/samples/csharp/getting-started/console-teleprompter/sampleQuotes.txt)
-file from the GitHub repository for this [sample](https://github.com/dotnet/docs/tree/master/samples/csharp/getting-started/console-teleprompter) into your project directory. 
-This will serve as the script for your
-application.
+The first feature to add is to read a text file, and display all that text to the console. First, let’s add a text file. Copy the [sampleQuotes.txt](https://raw.githubusercontent.com/dotnet/docs/master/samples/csharp/getting-started/console-teleprompter/sampleQuotes.txt) file from the GitHub repository for this [sample](https://github.com/dotnet/docs/tree/master/samples/csharp/getting-started/console-teleprompter) into your project directory. This will serve as the script for your
```

Comment:
The first feature to add is the ability to...
Also, no comma between "file" and "and"